incubator/gke-node-termination-handler/README.md

```diff
@@ -0,0 +1,19 @@
+# gke-node-termination-handler
+
+[gke-node-termination-handler](https://github.com/GoogleCloudPlatform/k8s-node-termination-handler)
+is a tool for gracefully draining preemptible GCP nodes before Google automatically shuts them down.
+
+## Installation
+We recommend installing gke-node-termination-handler in its own namespace.
+
+## Configuration
+Parameter | Description | Default
+--------- | ----------- | -------
+`image.repository` | Image repo | k8s.gcr.io/gke-node-termination-handler@sha256
+`image.tag` | Image tag | aca12d17b222dfed755e28a44d92721e477915fb73211d0a0f8925a1fa847cca
+`image.pullPolicy` | Image pull policy | IfNotPresent
+`daemonset.updateStrategy.type` | Daemonset update strategy type | RollingUpdate
+`resources.limits.cpu` | Daemonset CPU limit | 150m
```

Comment:
For future improvement, I think it's easier to use an entire resources block instead of each value independently.  If you do a `helm create <chart name>` you'll see the pattern I am referring to.